### ISSUE TITLE
Error is thrown on closed pipe in Windows 10

### ISSUE THREAD

**Issue by Nex-Otaku** [spiral/framework#242]
![unknown](https://user-images.githubusercontent.com/2260071/75964633-9856e580-5ed8-11ea-99f2-091457df9a78.png)

Steps to reproduce:
1. Install demo app on Windows 10
2. Start server "spiral -v -d serve"
3. Wait few hours...

Windows is closing pipes after inactivity, so fread() call from closed pipe returns empty string.
Now framework throws an error, trying to parse an empty string.

This is a standard OS behavior, and not a logic or any other code error. So it should be handled without exception.

I suggest to restart worker gracefully, without errors. Reason of restart should be logged to standard output.

Example:
`Received empty data from pipe. Seems like pipe has been closed by OS. Worker will be restarted.`

**Comment by wolfy-j** [comment]
Yeah, this is an annoying error we are going to address on worker end at roadrunner.
